src/components/messages/NewChat/index.tsx

```diff
@@ -94,12 +95,15 @@ const NewChat: React.FC = () => {
     [query, isInviting]
   )
 
+  console.log('Searchhh')
```

Comment:
Lets get rid of this before we forget :)

Reply:
Good catch :sweat: